src/utils/a11yPassThrough.js

```diff
@@ -0,0 +1,7 @@
+import cx from 'classnames';
+
+export default className =>
+	cx({
```

Comment:
This might read better using `includes`:

```
`visibility--a11yHide`: className.includes('visibility--a11yHide'),
`visibility--a11yShow`: className.includes('visiblity--a11yShow'),
```

I'd also leave a comment about why we have this.